ipn/ipnlocal/local.go

```diff
@@ -1241,21 +1241,24 @@ func (b *LocalBackend) populatePeerStatusLocked(sb *ipnstate.StatusBuilder) {
 				tailscaleIPs = append(tailscaleIPs, addr.Addr())
 			}
 		}
+		can, reason := b.taildropEligibility(p)
 		ps := &ipnstate.PeerStatus{
-			InNetworkMap:    true,
-			UserID:          p.User(),
-			AltSharerUserID: p.Sharer(),
-			TailscaleIPs:    tailscaleIPs,
-			HostName:        p.Hostinfo().Hostname(),
-			DNSName:         p.Name(),
-			OS:              p.Hostinfo().OS(),
-			LastSeen:        p.LastSeen().Get(),
-			Online:          p.Online().Get(),
-			ShareeNode:      p.Hostinfo().ShareeNode(),
-			ExitNode:        p.StableID() != "" && p.StableID() == exitNodeID,
-			SSH_HostKeys:    p.Hostinfo().SSH_HostKeys().AsSlice(),
-			Location:        p.Hostinfo().Location().AsStruct(),
-			Capabilities:    p.Capabilities().AsSlice(),
+			InNetworkMap:        true,
+			UserID:              p.User(),
+			AltSharerUserID:     p.Sharer(),
+			TailscaleIPs:        tailscaleIPs,
+			HostName:            p.Hostinfo().Hostname(),
+			DNSName:             p.Name(),
+			OS:                  p.Hostinfo().OS(),
+			LastSeen:            p.LastSeen().Get(),
+			Online:              p.Online().Get(),
+			ShareeNode:          p.Hostinfo().ShareeNode(),
+			ExitNode:            p.StableID() != "" && p.StableID() == exitNodeID,
+			SSH_HostKeys:        p.Hostinfo().SSH_HostKeys().AsSlice(),
+			Location:            p.Hostinfo().Location().AsStruct(),
+			Capabilities:        p.Capabilities().AsSlice(),
+			CanReceiveFiles:     can,
```

Comment:
Consider renaming to `TaildropTarget` or something that indicates this is for taildrop.